### DISCUSSION TITLE
Reversing downhill after going fast up.

### DISCUSSION THREAD

**Discussion by TekyTeky** [Novakasa/brickrail#153] [General]
I am progressing with the new Brickrail layout interweaving through and over the existing BAP.
The train goes up the winding incline, with the help of the TrixBrix rubber wheel rings and setting the blue and red block sensors to FAST then slow at the top.
Currently it is a dead end, so how do I reverse and not have it accelerate coming back down on the block I have set to fast.. is there are way for it be fast one way?
You may have said but we have had so many chats I can’t remember. 
![image](https://github.com/Novakasa/brickrail/assets/87248517/67538edc-600b-42b8-a44a-4f23feba319f)

**Comment by TekyTeky** [comment]
I just read this from a previous post.. but I don’t understand how you do it?

“ For the markers, I set the "marker speed" to fast when the train starts to go uphill, and set the speed to slow when it starts to go downhill. ”

I click the block sensor markers and change the speed in the right hand panel.

**Reply by Novakasa** [reply]
Similar to block settings, the track settings are also different depending on which "arrow" is visible when you click on the track and when it is selected. This way you can have different marker settings depending on which way the train is going.

**Reply by Novakasa** [reply]
So selecting this: 
![image](https://github.com/Novakasa/brickrail/assets/59559612/6c483aaf-12c0-4d01-84ce-77bb003cc53a)

will have different settings than selecting this:
![image](https://github.com/Novakasa/brickrail/assets/59559612/571bb7da-3e0e-4c30-a30b-53b23d032921)

**Reply by Novakasa** [reply]
And this also applies for sensor settings

**Reply by TekyTeky** [reply]
Oh.. ok.. got it.. I think :) 
I'll give it a try, thanks !!

It works great !!
